src/v/cluster/cluster_utils.h

```diff
@@ -56,11 +56,9 @@ class partition;
 
 /// Creates the same topic_result for all requests
 template<typename T>
-    requires requires(const T& req) {
-                 {
-                     req.tp_ns
-                     } -> std::convertible_to<const model::topic_namespace&>;
-             }
+requires requires(const T& req) {
```

Comment:
nit: a bit confused by the formatting.. why are they ill formatted in the first commit and undone here?